examples/oqmd/oqmd.ipynb

```diff
@@ -1,670 +1,777 @@
 {
- "cells": [
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# OQMD Mini Example"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from foundry import Foundry"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Instantiate Foundry from local file and load data from remote"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "f = Foundry()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "#need Globus to pull data\n",
-    "f = f.load('_test_foundry_oqmd_v1.1')\n",
-    "X,y = f.load_data()"
-   ]
+  "nbformat": 4,
```

Comment:
Instead of *Run to Create Dataframe and Metadata Locally without Fetch* I think it should be something like *Create Dataframe and Metadata Locally*, with a sentence or two like what I suggested for iris.ipynb